torchrl/envs/transforms/transforms.py

```diff
@@ -88,7 +88,7 @@ def new_fun(self, observation_spec):
             _specs = observation_spec._specs
             in_keys = self.in_keys
             out_keys = self.out_keys
-            for in_key, out_key in zip(in_keys, out_keys):
+            for in_key, out_key in zip(in_keys, out_keys, strict=True):
```

Comment:
We need to be compatible with python 3.8. PyTorch and TensorDict have a zip_strict that falls back onto zip(..., strict=True) for appropriate python versions

Reply:
Thanks for the pointer, I will make use of it!
Py3.8 is reaching end of life at the end of the month: https://devguide.python.org/versions/. What is the plan for supporting or dropping this version in the next torch and torchrl releases?

Reply:
I think this is a 3.10 feature so we have still a good 6 months until we can fully deprecate zip_strict

Reply:
I saw there was already some imports of private functions from `tensordict` so I did it too for `_zip_strict`.